app/Helpers/DatabaseHelper.php

```diff
@@ -64,22 +64,25 @@ public static function getResourcePathArray($id){
         return $res;
     }
 
-    public static function getObyxPoints($reason){
+    public static function getObyxPoints($reason) {
         $obyx = \DB::table('obyx')
             ->where('reason', '=', $reason)
             ->first();
 
         return $obyx->value;
     }
 
-    public static function getGameViewsMax(){
+    public static function getGameViewsMax() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->